conf/application.conf

```diff
@@ -242,7 +242,7 @@ Dev {
       }
 
       third-party-delegated-authority {
-        protocol = https
+        protocol = http
```

Comment:
I assumed you just delete the protocol line to match the other MS config? (not that I really care either way.)

Reply:
Oh yeah - brb